### PR TITLE
[EPMRPP-45078] Fix possible npe

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java [hunk 1]
```diff
@@ -135,18 +135,16 @@ public DeleteBulkRS deleteLaunches(DeleteBulkRQ deleteBulkRQ, ReportPortalUser.P
 	 * @param projectDetails {@link ReportPortalUser.ProjectDetails}
 	 */
 	private void validate(Launch launch, ReportPortalUser user, ReportPortalUser.ProjectDetails projectDetails) {
-		expect(launch, not(l -> l.getStatus().equals(StatusEnum.IN_PROGRESS))).verify(LAUNCH_IS_NOT_FINISHED,
+		expect(launch, not(l -> StatusEnum.IN_PROGRESS.equals(l.getStatus()))).verify(LAUNCH_IS_NOT_FINISHED,
 				formattedSupplier("Unable to delete launch '{}' in progress state", launch.getId())
 		);
-		if (user.getUserRole() != UserRole.ADMINISTRATOR) {
+		if (UserRole.ADMINISTRATOR != user.getUserRole()) {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java]: maybe equals ?

File: src/main/java/com/epam/ta/reportportal/core/item/impl/StartTestItemHandlerImpl.java [hunk 2]
```diff
@@ -180,7 +180,7 @@ private void handleRetries(Launch launch, TestItem item) {
 	 * @param launch         {@link Launch}
 	 */
 	private void validate(ReportPortalUser user, ReportPortalUser.ProjectDetails projectDetails, StartTestItemRQ rq, Launch launch) {
-		if (user.getUserRole() != UserRole.ADMINISTRATOR) {
+		if (UserRole.ADMINISTRATOR != user.getUserRole()) {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/item/impl/StartTestItemHandlerImpl.java]: maybe equals ?

File: src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java (hunk 1 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java]: fixed

File: src/main/java/com/epam/ta/reportportal/core/item/impl/StartTestItemHandlerImpl.java (hunk 2 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/core/item/impl/StartTestItemHandlerImpl.java]: fixed

File: src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java [hunk 3]
```diff
@@ -135,18 +135,16 @@ public DeleteBulkRS deleteLaunches(DeleteBulkRQ deleteBulkRQ, ReportPortalUser.P
 	 * @param projectDetails {@link ReportPortalUser.ProjectDetails}
 	 */
 	private void validate(Launch launch, ReportPortalUser user, ReportPortalUser.ProjectDetails projectDetails) {
-		expect(launch, not(l -> l.getStatus().equals(StatusEnum.IN_PROGRESS))).verify(LAUNCH_IS_NOT_FINISHED,
+		expect(launch, not(l -> StatusEnum.IN_PROGRESS.equals(l.getStatus()))).verify(LAUNCH_IS_NOT_FINISHED,
 				formattedSupplier("Unable to delete launch '{}' in progress state", launch.getId())
 		);
-		if (user.getUserRole() != UserRole.ADMINISTRATOR) {
+		if (UserRole.ADMINISTRATOR.equals(user.getUserRole())) {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java]: not equals

**Pull request merged**